active/001-tabular-msa.md

```diff
@@ -0,0 +1,82 @@
+---
+Feature name: tabular-msa
+Start date: 2015-08-14
+RFC PR: <leave empty>
+scikit-bio issue: <leave empty>
+---
+
+# Summary
+
+Replace the current experimental `skbio.alignment.Alignment` class with a
+`TabularMSA` class, providing an API for manipulating tabular (row-column)
```

Comment:
Ah, that is an excellent point about the labels. On the other hand, an adjacency matrix is still called a matrix even though there are labels. Agree re: MSA on the end

I don't have strong opinions on it, I just always thought of MSAs as matrices so just wanted to check :)